src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/scenarios/PaginationTable100Scenario.java

```diff
@@ -8,13 +8,15 @@
 import java.util.List;
 import java.util.Map;
 
-/** basic key-value table with 100 rows */
-public class KeyValueTable100Scenario extends TestDataScenario {
+/** basic table with 100 rows to test pagination */
+public class PaginationTable100Scenario extends TestDataScenario {
```

Comment:
make this a new scenario and keep the keyvalue one , and rename this to PartitionedKeyValueTableScenario, and insert data for three partitions , and change the number of rows so the last API page of data (which is 20 rows) is not full